test/test.c

```diff
@@ -24242,6 +24242,38 @@ test_JetStreamMgtConsumers(void)
         cfg.FilterSubject = "bar";
     }
 
+#define TIME_20350101 (2051251200L * 1000000000L)
+#define TIME_20350101_STR "\"2035-01-01T08:00:00Z\""
+
+    if (serverVersionAtLeast(2, 11, 0))
+    {
+        test("Add consumer (non durable, paused): ");
+        cfg.PauseUntil = TIME_20350101;
+        s = js_AddConsumer(&ci, js, "MY_STREAM", &cfg, NULL, &jerr);
+        testCond((s == NATS_ERR) && (jerr == JSStreamNotFoundErr) && (ci == NULL));
+        nats_clearLastError();
+
+        test("Verify config: ");
+        s = natsSubscription_NextMsg(&resp, sub, 1000);
+        testCond((s == NATS_OK) && (resp != NULL) && (strncmp(natsMsg_GetData(resp), "{\"stream_name\":\"MY_STREAM\","
```

Comment:
Assuming that what I mentioned above about checking consumer add request is correct: since we control the marshaling, it is likely that it will be in this order, etc.. but this test will have to be updated next time you add something to the consumer config. Would it not be enough to use `strstr` instead and look for `\"pause_until\":..`?

Reply:
Sort of not applicable after I moved the code, resolving